src/global/tracking/tracking.cc

```diff
@@ -3,6 +3,7 @@
 
 #include <DD4hep/Detector.h>
 #include <JANA/JApplication.h>
+#include <edm4eic/Track.h>
```

Comment:
Why did this stop suggesting `*Collection.h` as include?

Reply:
https://github.com/eic/EICrecon/blob/23e24712e949a752faeb1baecae95fe07541f7d3/.github/iwyu.imp#L26

Reply:
Is this something that needs to be addressed here?

Reply:
```suggestion
#include <edm4eic/TrackCollection.h>
```